wp-content/themes/core/assets/css/src/theme/typography/anchors.pcss

```diff
@@ -32,7 +32,8 @@ a {
  * ------------------------------------------------------------------------- */
 
 .anchor,
-.t-sink a {
+:where(.t-sink) a,
+:is(.editor-styles-wrapper) a {
```

Comment:
Originally, we discussed using `:where(.editor-styles-wrapper)` to handle admin styles. This would be ideal, but essentially has no effect on admin styles because WordPress core styles (usually from `common.css`) overwrite sink styles.

I like using `:is()` because we automatically get most sink styles working in the admin out of the box. The alternative, may be to revert this and just don't touch the admin at all.

Reply:
I'm a bit concerned that adding 2 or 3 top-level selectors will bloat our compiled CSS files even more. Would you be willing to compare the css file size between this branch and the default `main` branch, just to make sure it's not a significant jump?

Ideally, I think we'd find a way to automatically replace the `t-sink` selector with the `editor-styles-wrapper` selector on build of the admin css file(s). I'm not concerned with tackling that here though.

Reply:
@dpellenwood it looks like we have added about 12kb here (from 310kb to 322kb). If that seems acceptable for this iteration, I can backlog a ticket to improve upon this.

Reply:
Thanks for checking. You're good to merge this as-is. We already have some backlog tickets to look into breaking up our monolithic assets into smaller chunks. Those are bigger lifts down the road.